release-tool

```diff
@@ -961,14 +949,6 @@ build() {
         $xz -6 "${OUTPUT_DIR}/${tarball_name}"
     fi
 
-    if ! ${build_snapshot} && [ -e "${OUTPUT_DIR}/build-release" ]; then
-        logInfo "Cleaning existing build directory..."
-        rm -rf "${OUTPUT_DIR}/build-release" 2> /dev/null
-        if [ $? -ne 0 ]; then
-            exitError "Failed to clean existing build directory, please do it manually."
-        fi
-    fi
-
```

Comment:
Why removed?

Reply:
Because I added a different check above that errors out if the output directory already exists, so there is no need to clean the build directory anymore.

Reply:
The point of the snapshot is to NOT clean the directory.

Reply:
I could add the snapshot condition to the other check. Although I personally never use the release-tool for anything but building and bundling releases.

Reply:
Yah I use it to test the release tool or non compiling parts of the build process like packaging.

Reply:
Certain things don't even work properly without a clean directory. Such as building AppImages.